pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -941,9 +942,11 @@ public void unloadNamespaceBundlesGracefully() {
         unloadNamespaceBundlesGracefully(0, true);
     }
 
-    public void unloadNamespaceBundlesGracefully(int maxConcurrentUnload, boolean closeWithoutWaitingClientDisconnect) {
+    public void unloadNamespaceBundlesGracefully(
```

Comment:
I think this changes the behaviour before. we better take limit and timeUnit as parameter to create another method. Current adminAPI `BrokerBase.shutDownBrokerGracefully` will call this method as a persecond limit.

Reply:
This line https://github.com/apache/pulsar/pull/20766/files#diff-5ae71ad96b743188a5361f08676bd1026e51e4518b0a42e3d20735d5e5b5d994R556 ensures the compatibility for the pulsar-admin API